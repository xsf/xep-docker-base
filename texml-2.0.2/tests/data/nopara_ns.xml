<TeXML xmlns="http://getfo.sourceforge.net/texml/ns1">
One line.

Second line.

Third line.





All previous lines
should belong to one paragraph.
</TeXML>
