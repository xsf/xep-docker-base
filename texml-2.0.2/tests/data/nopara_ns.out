One line.
%
Second line.
%
Third line.
%
%
%
%
%
All previous lines
should belong to one paragraph.
